matlab/src/matlab/+arrow/recordBatch.m

```diff
@@ -16,7 +16,7 @@
 % permissions and limitations under the License.
 function rb = recordBatch(T)
     arguments
-        T table
+        T table= table.empty(0, 0)
```

Comment:
1. Could you add `{istable}` here?

Example: https://github.com/apache/arrow/blob/cbf1bb50875cc9716fccbb55b3d4cf9e8a33e4cd/matlab/src/matlab/%2Barrow/table.m#L21

This will ensure an error is thrown if non-`table` inputs are provided to the `arrow.recordBatch` function.

2. For clarity (and consistency with the `arrow.table` function), perhaps we should rename `T` to `matlabTable`?